templates/greet.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="ru">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+    <h1>
+        Приветствую тебя, {{ username }}
+        <br>
+        Вы ввели следующий email: {{ email }}
+    </h1>
+    <form action="/delite" method="post">
```

Comment:
Тогда шаблон можно изменить. Параметр `action` не нужен
```suggestion
    <form method="post">
```